pkg/deploy/deployment_che.go

```diff
@@ -112,10 +112,11 @@ func NewCheDeployment(cr *orgv1.CheCluster, cheImage string, cheTag string, cmRe
 										Scheme: corev1.URISchemeHTTP,
 									},
 								},
-								InitialDelaySeconds: 25,
-								FailureThreshold:    5,
+								InitialDelaySeconds: 60,
+								FailureThreshold:    10,
 								TimeoutSeconds:      5,
-							},
+								PeriodSeconds:       10,
```

Comment:
@davidfestal why `periodSeconds` is required for the `ReadinessProbe` ?

Reply:
Well, I just added the default value to make it clear what is being used. This is the value effectively added by OpenShift, that you can still find in the effective deployment, even when not putting it here.